src/core/src/energy/global/wolf.rs

```diff
@@ -172,10 +174,13 @@ impl GlobalPotential for Wolf {
 
     fn energy(&self, system: &System) -> f64 {
         let natoms = system.size();
-        let mut res = 0.0;
-        for i in 0..natoms {
+
+        (0..natoms).into_par_iter().map(|i|{
```

Comment:
As it seems to be a common pattern in the code, how about having a function: `range_par(Range<usize>)` to use in `range_par(0..natoms).map(|i| ...` ? Or even `range_par(natoms).map(|i| ...`?